change/@fluentui-react-charting-d99f14e6-dd3a-4f33-a8b3-9e28ea5bcfab.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33511&commitId=c24d9e57e18101c11b6e998b4b40036e3768b88f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.Bottom auto edge.chromium.png|2309| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>